CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
```

Comment:
You should follow the guidelines stablished by @etarancon for every repo in our organization